diff --git a/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake b/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
index 10e1ea16..cd6eec5f 100644
--- a/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
+++ b/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
@@ -22,6 +22,7 @@ cmake_minimum_required(VERSION 3.22)
 #*****************************************************************************************************************************************************
 
 set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_BINDIR}" "${CMAKE_INSTALL_LIBDIR}" "." "../lib")
+set(QT_DIR "/usr/lib/qt6")
 install(DIRECTORY "${QT_DIR}/qml" "${QT_DIR}/plugins"
         DESTINATION "${CMAKE_INSTALL_PREFIX}")
 
